### PR TITLE
2 Columns Redesign

### PR DESCRIPTION
**Pull request by andycarrell**

### CODE REVIEW

File: app/components/DosesDescriptionList/DosesDescriptionList.module.scss [hunk 1]
```diff
@@ -36,27 +16,23 @@
...
-}
 .dose-label {
+  text-transform: uppercase;
+  color: var(--text-secondary);
+
   font-size: 0.625rem; // 10px
   line-height: 0.875rem; // 14px
-  font-weight: 400;
-  text-transform: uppercase;
+  font-weight: 500;
```
Comment (andycarrell) [inline app/components/DosesDescriptionList/DosesDescriptionList.module.scss:25]: I put it in @finnclark hehe

File: app/components/DosesDescriptionList/DosesDescriptionList.tsx [hunk 2]
```diff
@@ -19,20 +20,21 @@ export const DosesDescription = ({
...
+  <div className="width-half">
+    <dt className={cx(styles["dose-label"], "margin-bottom-2xs")}>{term}</dt>
+    <dd className="flex flex-row justify-content-start align-items-center space-x-xs">
       <div
-        className={cx("data-text", { ["data-text-complete"]: hasMetTarget })}
+        className={cx(styles["percentage-value"], {
+          [styles["percentage-value-complete"]]: hasMetTarget,
+        })}
       >
-        {term}
```
Comment (jishaal) [inline app/components/DosesDescriptionList/DosesDescriptionList.tsx:27]: Term not being green anymore? 😢

File: app/components/DosesDescriptionList/DosesDescriptionList.module.scss (hunk 1 above)
Comment (finnclark) [reply app/components/DosesDescriptionList/DosesDescriptionList.module.scss:25]: yeah it looks better 👍

File: app/components/DosesDescriptionList/DosesDescriptionList.module.scss [hunk 3]
```diff
@@ -36,27 +16,23 @@
...
   letter-spacing: 0.05rem;
```
Comment (finnclark) [inline app/components/DosesDescriptionList/DosesDescriptionList.module.scss:26]: Tiny tweak

```suggestion
  letter-spacing: 0.03rem;
```

File: app/components/DosesDescriptionList/DosesDescriptionList.module.scss [hunk 4]
```diff
@@ -65,7 +41,14 @@
     line-height: 1.75rem; // 28px
   }
 }
+
+.percentage-value-complete {
+  color: var(--text-positive);
+}
+
 .vaccine-count {
+  color: var(--text-secondary);
```
Comment (finnclark) [inline app/components/DosesDescriptionList/DosesDescriptionList.module.scss:50]:
```suggestion
  color: var(--text-primary);
```

**Review by jishaal — COMMENTED** [review]
Is there a reference design anywhere in figma for this ;)

File: app/components/DosesDescriptionList/DosesDescriptionList.tsx [hunk 5]
```diff
@@ -41,45 +43,23 @@ export const DosesDescription = ({
...
-          </div>
-        ) : (
-          <div className="data-text">
-            <strong>
-              {numberFormatter.format(dhbData.secondDosesTo90Percent)}
-            </strong>{" "}
-            left
-          </div>
-        )}
+      <dd className={cx(styles["vaccine-count"], "margin-top-px")}>
```
Comment (finnclark) [inline app/components/DosesDescriptionList/DosesDescriptionList.tsx:61]: Same here

```suggestion
      <dd className={cx(styles["vaccine-count"])}>
```

File: app/components/DosesDescriptionList/DosesDescriptionList.tsx [hunk 6]
```diff
@@ -41,45 +43,23 @@ export const DosesDescription = ({
...
-          </div>
-        ) : (
-          <div className="data-text">
-            <strong>
-              {numberFormatter.format(dhbData.firstDosesTo90Percent)}
-            </strong>{" "}
-            left
-          </div>
-        )}
+      <dd className={cx(styles["vaccine-count"], "margin-top-px")}>
```
Comment (finnclark) [inline app/components/DosesDescriptionList/DosesDescriptionList.tsx:52]: Does this 1px do something functionally? Not desired from a design POV
```suggestion
      <dd className={cx(styles["vaccine-count"])}>
```

File: app/components/DosesDescriptionList/DosesDescriptionList.module.scss [hunk 7]
```diff
@@ -36,27 +16,23 @@
...
+  font-weight: 500;
   letter-spacing: 0.05rem;
   @media (min-width: $breakpoint-s) {
     font-size: 0.75rem; // 12px
     line-height: 1rem; // 16px
   }
 }
+
 .percentage-value {
+  color: var(--text-secondary);
```
Comment (finnclark) [inline app/components/DosesDescriptionList/DosesDescriptionList.module.scss:34]:
```suggestion
  color: var(--text-primary);
```

**Review by finnclark — CHANGES_REQUESTED** [review]

File: app/components/DosesDescriptionList/DosesDescriptionList.tsx (hunk 6 above)
Comment (andycarrell) [reply app/components/DosesDescriptionList/DosesDescriptionList.tsx:52]: Nothing functional, will remove!

File: app/components/DosesDescriptionList/DosesDescriptionList.tsx (hunk 2 above)
Comment (andycarrell) [reply app/components/DosesDescriptionList/DosesDescriptionList.tsx:27]: See "top right" of the figma file

image

File: app/styles/spacing.scss [hunk 8]
```diff
@@ -2,6 +2,7 @@
 
 $space-map: (
   "none": $space-none,
+  "px": 1px,
```
Comment (andycarrell) [inline app/styles/spacing.scss:5]: Do we still need? Might be useful at some point I guess

Comment (finnclark) [reply app/styles/spacing.scss:5]: My preference is to not have stuff we aren't really using, unless it's part of a larger system. But up to you :)

File: app/components/DosesDescriptionList/DosesDescriptionList.tsx (hunk 2 above)
Comment (jishaal) [reply app/components/DosesDescriptionList/DosesDescriptionList.tsx:27]: Ah cool ty!

**Review by jishaal — APPROVED** [review]

**Pull request merged**